schemas/mod-data-import-converter-storage/mapping-profile-detail/mappingRule.json

```diff
@@ -32,9 +32,10 @@
       "$ref": "repeatableFieldActions.json"
     },
     "subfields": {
-      "description": "List of repeatable subfields",
+      "description": "List of fields for repeatable subfield",
+      "javaType": "java.util.List<org.folio.rest.jaxrs.model.RepeatableSubfieldMapping>",
```

Comment:
Do we need to specify "javaType" in property definition for schema which already contains "javaType"?